homeassistant/components/jvc_projector/entity.py

```diff
@@ -0,0 +1,44 @@
+"""Base Entity for the jvc_projector integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    DOMAIN as JVC_DOMAIN,
+    MANUFACTURER as JVC_MANUFACTURER,
+    NAME as JVC_NAME,
+)
```

Comment:
I'm not sure adding `JVC_` prefix adds anything here.

Reply:
I meant on all three

Reply:
Oops, totally glazed over the others. Fixed.